packages/protocol/test/common/integration.ts

```diff
@@ -166,7 +191,13 @@ contract('Integration: Exchange', (accounts: string[]) => {
       await exchange.exchange(sellAmount, minBuyAmount, true)
       const finalGold = await goldToken.balanceOf(accounts[0])
 
-      assert.isTrue(finalGold.lt(originalGold))
+      const block = await web3.eth.getBlock('latest')
```

Comment:
Why was this section added?

Reply:
The addition of tests caused the block reward to go to `accounts[0]`, throwing the gold balance comparison off. The added code guards against this.